README.md

```diff
@@ -59,6 +59,88 @@ style guide that you will not find in the GitHub one.
 
 ## Standards ##
 
+### Naming Convention ###
+
+With Swift 3, Apple updated the Swift [API design guidelines](https://swift.org/documentation/api-design-guidelines/) to promote clarity over brevity.
+
+Capitalize all type names with camel case, while function and variable names start with a lower case. Avoid acronyms and abbreviations for clarity and readability.
+
+#### Protocols ####
+
+Protocols describing something should be a noun: `Collection`, `Element`. Protocols describing an ability should end with “ing” or “able”: `Evaluatable`, `Printable`, `Formatting`.
```

Comment:
I don't get what the first sentence means. `A protocol should BE a noun`? Could be more clear and maybe rephrase? Also use `for example` or `e.g.`